airbyte-integrations/connectors/destination-s3/src/test/java/io/airbyte/integrations/destination/s3/tamplate/S3FilenameTemplateManagerTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.s3.tamplate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mockStatic;
+
+import io.airbyte.integrations.destination.s3.template.S3FilenameTemplateManager;
+import io.airbyte.integrations.destination.s3.template.S3FilenameTemplateParameterObject;
+import java.io.IOException;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+
+class S3FilenameTemplateManagerTest {
+
+  private final S3FilenameTemplateManager s3FilenameTemplateManager = new S3FilenameTemplateManager();
+
+  @Test
+  @DisplayName("Should replace the date placeholder with the current date in the format YYYY-MM-DD")
+  void adaptFilenameAccordingSpecificationPatternWithDefaultConfigShouldReplaceDatePlaceholderWithCurrentDateInTheFormatYYYY_MM_DD()
```

Comment:
ehm... if the name of the method is longer than the @DisplayName description that should be a red flag. But seriously, maybe rename this to something like `testDatePlaceholder` and rename the next test to `testTimestampPlaceholder`

Reply:
> ehm... if the name of the method is longer than the @DisplayName description that should be a red flag. But seriously, maybe rename this to something like `testDatePlaceholder` and rename the next test to `testTimestampPlaceholder`

Fixed